sqldb/sqlc/queries/amp_invoices.sql

```diff
@@ -1,79 +1,67 @@
--- name: InsertAMPInvoicePayment :exec
-INSERT INTO amp_invoice_payments (
-    set_id, state, created_at, settled_index, invoice_id
+-- name: UpsertAmpInvoice :execresult
+INSERT INTO amp_invoices (
+    set_id, state, created_at, invoice_id
 ) VALUES (
-    $1, $2, $3, $4, $5
-); 
+    $1, $2, $3, $4
+) ON CONFLICT (set_id, invoice_id) DO NOTHING;
 
--- name: SelectAMPInvoicePayments :many
-SELECT aip.*, ip.*
-FROM amp_invoice_payments aip LEFT JOIN invoice_payments ip ON aip.settled_index = ip.id
```

Comment:
I think this query is still relevant, as it's how you can get all the payments to a given AMP invoice. Otherwise, IIUC with the new diff, you'd need to create some new composite query by first querying for the invoice, then all the HTLCs, and sum all of those. Though I guess the old AMP payment table didn't have the amount there either...